### PR TITLE
Specify namespace in ResourceUsageSummary measurement

### PR DESCRIPTION
**Pull request by sozercan**
Adds a parameter for namespace to be used for ResourceUsageSummary measurement

### CODE REVIEW

File: clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go [hunk 1]
```diff
@@ -106,13 +106,10 @@ func NewResourceUsageGatherer(c clientset.Interface, host string, port int, prov
...
 		})
 	} else {
-		// Tracks kube-system pods if no valid PodList is passed in.
 		var err error
-		if pods == nil {
-			pods, err = c.CoreV1().Pods("kube-system").List(context.TODO(), metav1.ListOptions{})
-			if err != nil {
-				return nil, fmt.Errorf("listing pods error: %v", err)
-			}
+		pods, err := c.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{})
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/util/gatherers/container_resource_gatherer.go:110]: remove "var err error" above - it's no longer needed

**Comment by wojtek-t** [comment]
@sozercan - please squash commits

**Comment by sozercan** [comment]
@wojtek-t thanks, done

**Pull request merged**